### ISSUE TITLE
Config file search only happens under `projectRootPath` 

### ISSUE THREAD

**Issue by mhegazy** [Microsoft/TypeScript#21209]
* Create folder A, with tsconfig.json and a single file a.ts
* Open VScode using `code ./A`
* try to open a file from a different project, say `src/compiler/checker.ts`

**Expected behavior:**

the tsconfig.json for `src/compiler/checker.ts` is found and is loaded correctelly

**Actual behavior:**

the tsconfig.json is not loaded, resulting in errors for missing types.

**Comment by sheetalkamat** [comment]
Isnt this by design... Project root is suppose to be the directory till which we can lookup the config file which vscode sets as the directory of opened folder..

**Comment by mhegazy** [comment]
yes and no. we should be looking in the projectRoot first. but if it fails, we should go back to the normal lookup process.
